src/pages/Home/subcomponents/_CityMap.scss

```diff
@@ -691,4 +705,11 @@
 
   .mapboxgl-ctrl-geocoder--icon-close {
     margin-top: 4px;
+  }
+
+  .city-map-readonly {
+    .city-new-side-menu {
+      top: 0;
+      height: 100%;
+    }
   }
```

Comment:
New line for this file please 👍

Reply:
Sorry, what do you mean?

Reply:
Ah I was talking about inserting a new line at the end of this file.